# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1479050468 18000
#      Sun Nov 13 10:21:08 2016 -0500
# Branch stable
# Node ID 1c67938a15f10305ce080f8100507f1de4259c78
# Parent  708a4fcb73820484a3af94dd4d18453c4d9b29c9
Added tag release-4-2-0 for changeset 708a4fcb7382

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -100,8 +100,9 @@ 0cfcc178432bfc7cfdef5dca96e74ec5884ba752
 a58d5e6df908199c4aea83f78077b81289d2a0ad rc-4-0-1-3
 2b66f30a0de7a09801c2a3566021cde7e194d3d3 rc-4-0-1-4
 5162cdfb56d8e31ad00dc260a26d1c92c69c2acf release-4-0-1
 acaf1e137c5e6f201a6689a99a6568d0dfe9e9ed release-4-0-2
 00f7b278defd1b6b911cde1ff20390fb298f4c5b release-4-0-3
 b8eeca55fa8113853a19b1b1ab4d0bb6471dea18 rc-4-2-0-1
 9d50627498141d150755d18ab8e71a3220e40147 rc-4-2-0-2
 69ac19073ae630b276ff7788b78f72782f281657 rc-4-2-0-4
+708a4fcb73820484a3af94dd4d18453c4d9b29c9 release-4-2-0
